pyproject.toml

```diff
@@ -11,48 +11,49 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-alembic = "^1.7.7"
-authlib = "^0.11"
-authutils = "6.1.0"
+python = ">=3.7,<4.0"
+alembic = "1.8.1"
+authlib = "<=0.14.2"
+authutils = "4.0.0"
 bcrypt = "^3.1.4"
 boto3 = "~1.9.91"
 botocore = "^1.12.253"
 cached_property = "^1.5.1"
 cdiserrors = "<2.0.0"
 cdislogging = "^1.0.0"
-cdispyutils = "^2.0.1"
-cryptography = ">=3.2"
-flask = ">=1.1.1"
+cdispyutils = "^1.0.4"
+cryptography = "==2.8"
+flask = "^1.1.1"
 MarkupSafe = "<2.0"
 flask-cors = "^3.0.3"
 flask-restful = "^0.3.6"
 flask_sqlalchemy_session = "^1.1"
 email_validator = "^1.1.1"
-gen3authz = "^1.5.1"
-gen3cirrus = "^2.0.0"
+gen3authz = "^1.0.4"
+gen3cirrus = "^1.3.0"
 gen3config = "^0.1.7"
 gen3users = "^0.6.0"
 markdown = "^3.1.1"
 paramiko = "^2.6.0"
-psycopg2 = "^2.8.3"
+psycopg2 = "^2.8.6"
 pyjwt = "^1.5.3"
 python_dateutil = "^2.6.1"
 python-jose = "^2.0.2"
 pyyaml = "^5.1"
-requests = ">=2.18.0"
+requests = "^2.18.0"
 retry = "^0.9.2"
 sqlalchemy = "^1.3.3"
 marshmallow-sqlalchemy = "0.24.2"
-werkzeug = "^1.0.0"
+werkzeug = "^0.16.0"
 html2text = "^2020.1.16"
 pycryptodome = "3.9.8"
+hubspot-api-client = "^6.0.0"
```

Comment:
Aside for adding this new dependency, please revert the pyproject.toml dependencies version to the previous state.